src/sdg_hub/flows/qa_generation/document_grounded_qa/enchance_multi_summary_qa/extractive_summary/flow.yaml

```
@@ -0,0 +1,117 @@
+      - "icl_response_1"
+      - "icl_query_2"
+      - "icl_response_2"
+      - "icl_query_3"
+      - "icl_response_3"
+    description: "Input dataset should contain documents with text content and domain classification. Each document should be substantial enough for meaningful question generation (minimum 100 words recommended). The flow generates three types of summaries: detailed (n=20), extractive (n=10), and key facts (n=50), each producing corresponding QA pairs designed to help LLMs internalize document knowledge for knowledge tuning."
+
+  output_columns:
+    - "summary_detailed"
+    - "question"
+    - "response"
+    - "summary_extractive" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)